src/Kentico.Xperience.Lucene/Admin/Client/package.json

```
@@ -27,6 +27,7 @@
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "6.7.0",
+    "babel-eslint": "^10.1.0",
```

```suggestion
    "babel-eslint": "10.1.0",
```

Is this dependency needed for something? If so, it should be an explicit version number

---

This dependency was replaced with `@babel/eslint-parser` since `babel-eslint` is deprecated.